Cargo.toml

```diff
@@ -14,8 +14,3 @@ default-members = [
     "price-estimator",
     "pricegraph",
 ]
-
-[patch.crates-io]
-# Needed to make coverage work on MacOS (https://github.com/servo/core-foundation-rs/pull/357)
-core-foundation = { git = "https://github.com/servo/core-foundation-rs", tag="v0.20.1"}
-core-foundation-sys = { git = "https://github.com/servo/core-foundation-rs", tag="v0.20.1"}
```

Comment:
Removed this because I got a warning when building
> ... was not used in the crate graph.

This probably means the dependency is no longer used so overriding it doesn't make sense.